rebar.config

```diff
@@ -0,0 +1,52 @@
+%%% -*- mode: erlang; -*-
+{erl_opts,
+ [
+  warn_unused_vars,
+  warn_export_all,
+  warn_shadow_vars,
+  warn_unused_import,
+  warn_unused_function,
+  warn_bif_clash,
+  warn_unused_record,
+  warn_deprecated_function,
+  warn_obsolete_guard,
+  strict_validation,
+  warn_export_vars,
+  warn_exported_vars,
+  warn_missing_spec,
+  warn_untyped_record,
+  debug_info,
+  {d, rebar}
+ ]
+}.
+
+{erl_first_files, ["src/ktn_recipe.erl"]}.
+
+{deps,
+ [
```

Comment:
Yes. I was misreading the source, because I only found those libraries mentioned in tests. Sounds like most of what I put under the "test" profile should actually be in the main section.

I break the circular dependency using `override` to stomp on elvis_core's declared deps. It's not ideal, since it means listing lager and zipper's version number in two places.